app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -43,7 +43,9 @@
             "external": "false",
             "text": "GOV.UK",
             "section": "Logo",
-            "index": 0,
+            "index": {
+              "index_section": 0,
```

Comment:
One to check with the PA's: should this be `index_link`?

Reply:
Have asked, will update when I have an answer, thanks for raising.